File: 167-two-sum-ii-input-array-is-sorted/two-sum-ii-input-array-is-sorted.py
class Solution(object):
    def twoSum(self, numbers, target):
       d={}
       sum=0
       for i in range(len(numbers)):
        sum=target-numbers[i]
        if sum in d:
            return [d[sum]+1,i+1]
        else:
            d[numbers[i]]=i
       return -1

File: 167-two-sum-ii-input-array-is-sorted/two-sum-ii-input-array-is-sorted.java
class Solution {
    public int[] twoSum(int[] numbers, int target) {
        int i=0;
        int k=numbers.length-1;
        while(i<k){
           if(numbers[i]+numbers[k]==target)
             return new int[]{i+1,k+1};
           else if(numbers[i]+numbers[k]<target)
                  i++;
           else
              k--; 
        }
        return new int[]{-1,-1};
    }
}